docs/getting-started/installing-regression-games.mdx

```diff
@@ -4,6 +4,115 @@ sidebar_label: 'Installing Regression Games'
 
 # Installing Regression Games
 
-import Placeholder from '../partials/_placeholder.mdx';
+This page covers everything you need to know to add the Regression Games SDK to your Unity project and start exploring its features.
 
-<Placeholder />
\ No newline at end of file
+## Add the Regression Games SDK to Your Project
+
+Our core package is [publicly available on GitHub](https://github.com/Regression-Games/RGUnityBots).
+Open your Unity project and navigate to `Window` > `Package Manager`. From here, find the `Add package from git URL` and paste the following URL:
+
+```
+https://github.com/Regression-Games/RGUnityBots.git?path=src/gg.regression.unity.bots#v0.0.27
+```
+
+:::info
+- This package utilizes TextMeshPro. If you are prompted by Unity to add TextMeshPro assets to your project, please add them.
+- This package installs the [Newtonsoft Json](https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.1/manual/index.html) dependency for serializing/deserializing Json message payloads.
+- If you get a note about Unity's new input system, click **Yes** and allow the editor to restart.
+:::
+
+![Add the Regression Games core SDK to your Unity project](img/installing-regression-games/installing-sdk.gif)
+
+## Add Extension Packages to Your Project
+
+We provide the following extensions for our core SDK to support common project dependencies.
+Each of these can be added to your project using a git url, similar to core package installation.
+
+### Animation Rigging Extension
+
+Adds support for Unity Animation Rigging.
+
+```
+https://github.com/Regression-Games/RGUnityBots?path=src/gg.regression.unity.bots.animation.rigging#v0.0.27
+```
+
+### Entity Component System (ECS) Extension
+
+Enables game state collection of Unity ECS Entities and their Components.
+Without this extension, the core SDK can still capture GameObject data in projects using ECS but will have no access to ECS Entities.
+Depending on your project and its needs, this extension may not be necessary, but we highly recommend including it
```

Comment:
Do recommend it for projects that are not using ECS as well? Reading this that was my question.